README.md

```diff
@@ -19,6 +19,11 @@ If that doesn't work:
  * [re-try with `LIBSSH2_SYS_USE_PKG_CONFIG=whatever`](https://github.com/nabijaczleweli/cargo-update/issues/129#issuecomment-599269219),
  * [install OpenSSL via `brew`, and re-try with `LDFLAGS="-L/usr/local/opt/openssl@1.1/lib" CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"`](https://github.com/nabijaczleweli/cargo-update/issues/123),
  * [verify that you don't `openssl` installed twice via `brew`](https://github.com/nabijaczleweli/cargo-update/issues/121#issuecomment-570673813),
+ * [consult the `openssl` crate's instructions](https://github.com/sfackler/rust-openssl#macos),
```

Comment:
The link doesn't go anywhere.

Reply:
Sorry copied from https://github.com/alexcrichton/ssh2-rs I will remove the anchor